Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8606,8 +8732,10 @@ def setup_proxy():
 
 
 def main():
-    LOG('command is {}'.format(str(demisto.command())))
+    demisto.debug(f"command is {demisto.command()}")
     command = demisto.command()
+    args: Dict[str, str] = demisto.args()
+    should_error = argToBoolean(demisto.params().get('with_error', False))
```

Comment:
Honestly, not sure we need `should_error` at all anymore since we return an error entry (instead of using `return_error`) in all command flows. The only usage it has at this point is to return a warning entry instead of an error entry which I'm not sure is needed as well.